paasta_tools/paasta_maintenance.py

```diff
@@ -472,6 +474,29 @@ def is_safe_to_kill(hostname):
     return is_host_drained(hostname) or is_host_past_maintenance_start(hostname)
 
 
+def is_hostname_local(hostname):
+    return hostname == 'localhost' or hostname == getfqdn() or hostname == gethostname()
+
+
+def is_safe_to_drain(hostname):
+    """Checks if a host has healthy tasks running locally that have low
+    replication in other places
+    :param hostname: hostname to check
+    :returns: True or False
+    """
+    if not is_hostname_local(hostname):
+        print (
+            "Due to the way is_safe_to_drain is implemented, it can only work "
```

Comment:
Technically all that information is available over the network, though our code for retrieving it is all centered on the current host. I'm fine with this for now; we can modify later if we find a need for is_safe_to_drain remotely.